terra/core/settings.py

```diff
@@ -635,6 +698,8 @@ def default(self, obj):
       if obj._wrapped is None:
         raise ImproperlyConfigured('Settings not initialized')
       return TerraJSONEncoder.serializableSettings(obj._wrapped)
+    # elif isinstance(obj, datetime):
+    #   return str(obj)
```

Comment:
It's an idea I'm playing with for in the future. Right now most things in terra.settings are json serializable. The exception being features that are evaluated into json serializable objects at dump time.

I was trying to add `terra.settings.terra.start_time` as a `datetime` object. And trying to figure out how this can fit into the rest of terra. It didn't work with 5 minutes worth of effort, so I put it off for "later".

I will probably have to make a "wrapper/proxy" class, that can be JSON serialized (using pickle) and unserialized (using unpickle), so that the final result is the object again.

This feature would be for default set objects only, not currently intended for `config.json` set values, that would just be complicated.